api/src/champion-mastery/controller.ts

```diff
@@ -2,7 +2,8 @@ import type { API } from 'lambda-api';
 import { getChampionMastery } from './service';
 
 export function championMasteryApi(api: API) {
-  api.get('/', (req, res) => {
-    return res.status(200).send(getChampionMastery());
+  api.get('/', async (req, res) => {
+    const getChampionMasteryResponse = await getChampionMastery();
+    return res.status(200).send(getChampionMasteryResponse.slice(0, 5));
```

Comment:
@reishaleem Adding a note to #7, but `send` is `void` so we shouldn't return it. But I also don't like sending the response from here instead of the handler, seems weird.

I also think this is sending as a string, will need to look into that

Reply:
Ok